{"changes": {"state": {"new": "resolved", "old": "new"}}, "links": {"self": {"href": "data/repositories/osrf/vrx/issues/152/changes/56883737.json"}, "html": {"href": "#!/osrf/vrx/issues/152#comment-56883737"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/vrx/issues/152.json"}}, "type": "issue", "id": 152, "repository": {"links": {"self": {"href": "data/repositories/osrf/vrx.json"}, "html": {"href": "#!/osrf/vrx"}, "avatar": {"href": "data/bytebucket.org/ravatar/{e278ca38-7edb-4e62-b785-46dff5617d98}ts=2274605"}}, "type": "repository", "name": "vrx", "full_name": "osrf/vrx", "uuid": "{e278ca38-7edb-4e62-b785-46dff5617d98}"}, "title": "Runtime Error During World Generation"}, "created_on": "2020-04-10T04:04:10.294226+00:00", "user": {"display_name": "Brian Bingham", "uuid": "{9cdea257-248f-486e-8858-49d8485a916f}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B9cdea257-248f-486e-8858-49d8485a916f%7D"}, "html": {"href": "https://bitbucket.org/%7B9cdea257-248f-486e-8858-49d8485a916f%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:6c52d6e5-77e0-41a8-8b6e-cb9462f4d5f1/7a690884-fc47-4bca-9c2b-baecee4d3dc9/128"}}, "nickname": "brian_bingham", "type": "user", "account_id": "557058:6c52d6e5-77e0-41a8-8b6e-cb9462f4d5f1"}, "message": {"raw": "Marking as resolved for now - reopen if necessary.", "markup": "markdown", "html": "<p>Marking as resolved for now - reopen if necessary.</p>", "type": "rendered"}, "type": "issue_change", "id": 56883737}